tests/components/aurora/test_config_flow.py

```diff
@@ -1,117 +1,99 @@
 """Test the Aurora config flow."""
 
-from unittest.mock import patch
+from unittest.mock import AsyncMock
 
 from aiohttp import ClientError
+import pytest
 
-from homeassistant import config_entries
-from homeassistant.components.aurora.const import DOMAIN
+from homeassistant.components.aurora.const import CONF_THRESHOLD, DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
 from homeassistant.core import HomeAssistant
 from homeassistant.data_entry_flow import FlowResultType
 
 from tests.common import MockConfigEntry
+from tests.components.aurora import setup_integration
 
 DATA = {
-    "latitude": -10,
-    "longitude": 10.2,
+    CONF_LATITUDE: -10,
+    CONF_LONGITUDE: 10.2,
```

Comment:
Is there a way to have a single set of constants that are used here and in conftest.py:49?